driverService/src/test/java/com/modsen/driverservice/util/TestData.java

```diff
@@ -0,0 +1,200 @@
+package com.modsen.driverservice.util;
+
+import com.modsen.driverservice.dto.CarRequestDto;
+import com.modsen.driverservice.dto.CarResponseDto;
+import com.modsen.driverservice.dto.DriverRequestDto;
+import com.modsen.driverservice.dto.DriverResponseDto;
+import com.modsen.driverservice.dto.UserRatingDto;
+import com.modsen.driverservice.model.Car;
+import com.modsen.driverservice.model.Driver;
+
+import java.util.List;
+
+public class TestData {
```

Comment:
Для утильных классов:
```suggestion
@NoArgsConstructor(access = AccessLevel.PRIVATE)
public final class TestData {
```